src/Pretix/OrderHelper.php

```diff
@@ -86,12 +88,16 @@ public function getOrder($organizer, $event, $orderCode) {
     foreach ($allQuotas as $quota) {
       foreach ($quota->getItems() as $itemId) {
         try {
-          $availability = $this->pretixClient->getQuotaAvailability($event,
-            $quota);
+          $availability = $this->pretixClient->getQuotaAvailability(
+                $event,
+                $quota
+            );
         }
         catch (\Exception $exception) {
-          throw $this->clientException('Cannot get quota availability',
-            $exception);
+          throw $this->clientException(
+                'Cannot get quota availability',
+                $exception
+                    );
```

Comment:
The coding standards may not check/require this, but please be consistent in how code is indented (cf. lines 69–72).

Reply:
Fixed